src/lib/message/BUILD.gn

```diff
@@ -45,6 +45,7 @@ static_library("message") {
   ]
 
   public_deps = [
+    "${chip_platform_root}",
```

Comment:
I think all these dependencies should be left alone. //src/platform can add the external platform as a dependency instead of being replaced outright.